### ISSUE TITLE
Accessibility - Toast isn't announced when user is using the talkback on Android

### ISSUE THREAD

**Issue by aljidy** [arnnis/react-native-fast-toast#178]
### Current behaviour

On Android, toasts are not announced by Talkback(Screenreader).  This means that users using this accessibility feature/users with a visual impairment won't know that a toast was displayed or what it said.

_Untested on iOS_

### Expected behaviour

When the toast is shown, a screenreader announcement should be made. 

_This could be an opt out functionality if needed_
### Code sample

The sample Android app. 

### What have you tried

This seems like an oversight, so seems like an issue in the module itself. 

### Your Environments

Not environment dependent.
